fiatconnect-api.md

```diff
@@ -1663,7 +1663,8 @@ An enum listing the types of fiat currencies supported by FiatConnect.
 ```
 [
 	`USD`,
-	`EUR`
+	`EUR`,
+ 	`BRL`
```

Comment:
@rafapolo I think this should be consistent with the change you added to the swagger spec where the currency was referred to by `REAL`